package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "node-rps-andrea",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Maybe you can add the `prettier` dev dependency and commands to check and fix the formatting as used in our projects.

Reply:
done in step 1 :)